### ISSUE TITLE
tweak: Alumiumn Ore/Ingot

### ISSUE THREAD

**Issue by artdude543** [Soartex-Modded/Modded-1.12.x#2]
The ingot and/or ore needs tweaking somewhat with some of the mods to help with the differences from Iron as it's close. Also, some people have issues seeing the changes. CC: @Darkosto

**Comment by zISKo666** [comment]
Thinking about this we should unify all ingots, ores, dusts and blends.

- Aluminium (Aluminum/Bauxite) ingots/ores/dusts
- Copper ingots/ores/dusts
- Tin ingots/ores/dusts
- Lead ingots/ores/dusts
- Uranium ingots/ores/dusts
- Bronce ingots/blends
- ... etc ...

There are other items also, like plates, gears and more.

**Comment by artdude543** [comment]
Yeah, that is true and we could use the same across the board. I have re-done all the item textures in TE so we could use them as the master and copy over to the other mods?

**Comment by ScottKillen** [comment]
There is actually another (better?) way to do it in 1.12 rather than copying. Now we can change the models so they actually use the same physical textures. As an example, [the Lithos pack](https://minecraft.curseforge.com/projects/lithos-core-32x-1-6-1-11-complete) does this beautifully.

**Comment by artdude543** [comment]
Yeah, I've seen other packs do it; but due to our patcher systems and people generally, download single patches rather than the full download. We'd need the files to be in all the individual mods folders.

It's a little bit more effort but we can make a new repo which contains all the master files for ingots/ore etc... which then people can use for reference/copy paste. Unless we make a patch which is all the masters and then link to that using the model JSON's but that would require an update to the backend scripts etc... But could be an idea.

**Comment by artdude543** [comment]
Okay, I've re-written the backend script to handle this only took 3 days lol xD But I've started working on the master files and/or changes and will be pushing to this repo [Soartex Ores](https://github.com/Soartex-Modded/Soartex-Ores) and have started making the model files for the mods already in this repo. So I'll push all the changes at once.

**Comment by leagris** [comment]
Another option would be using symbolic links

**Comment by leagris** [comment]
The alternative is to use blue aluminium from Gregtech 2-4 assets:

#### Gregtech Aluminium assets:

- ##### Ore  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.5.x/master/GregTech/mods/GregTech_Addon/textures/blocks/tile.Ore_Stone/12.png)
- ##### Dust  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_Dusts/18.png)
- ##### Small Dust  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_SmallDusts/18.png)
- ##### Tiny Dust  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_TinyDusts/18.png)
- ##### Ingot  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_Materials/18.png)
- ##### Nugget  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_Nuggets/18.png)
- ##### Plate  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_Materials/75.png)
- ##### Rod  
  ![](https://raw.githubusercontent.com/Soartex-Modded/Modded-1.6.x/master/GregTech/assets/gregtech_addon/textures/items/item.GT_Materials/107.png)
